### ISSUE TITLE
[Bug] n.basename is not a function error when uploading png image

### ISSUE THREAD

**Issue by pedropacheco-hotmart** [abramenal/cypress-file-upload#250]
### Current behavior:

As I'm running a Cypress test with this package, I'm getting an `n.basename is not a function` error when I call the attachFile function from a hidden input of type file trying to upload a png image. Already tried invoking the `show` method so it's visible before attaching without success.

<img width="398" alt="Captura de Tela 2021-01-13 às 16 37 20" src="https://user-images.githubusercontent.com/71897096/104505929-6d8ef080-55c3-11eb-97fa-db1c663ad78a.png">

### Desired behavior:

The file should be uploaded without issues

### Steps to reproduce: (app code and test code)

- create a react project with a HTML hidden input of type file and the attribute `data-cy="file-input"`
- add a png image named testImage.png to the fixture folder
- add `cy.get('[data-cy="file-input"]').attachFile('testImage.png')` to a cypress spec file
- run the created spec

### Versions

Cypress: 6.2.1
Package: 4.1.1

**Comment by abramenal** [comment]
Hi @pedropacheco-hotmart 
Thanks for submitting the issue!

The only usage of `basename` can be found in this file: https://github.com/abramenal/cypress-file-upload/blob/main/src/helpers/getFixtureInfo.js#L9

So I assume this might be related to your Node.js version, as it's internal Node package.
Can you please share that?

**Comment by pedropacheco-hotmart** [comment]
Hello there @abramenal! My current node version is `v14.12.0`.

**Comment by Ckrill** [comment]
I started seeing a similar error (`TypeError: o.default.basename is not a function`) after upgrading to  Gatsby 3 (3.0.4), and therefore Webpack 5 (5.16.0).
Could it be that the Webpack upgrade caused this?

**Comment by abramenal** [comment]
Hey peeps, can anyone check if this issue is reproducible in this package's version?
In your package.json:
```
"cypress-file-upload": "github:abramenal/cypress-file-upload#feature/fix-bundle",
```

It tooks the code from this PR's branch: https://github.com/abramenal/cypress-file-upload/pull/285

**Comment by Ckrill** [comment]
Hi @abramenal 
I am still seeing my error message, but I will try to upgrade to Cypress 7 and test if it persists after that.

**Comment by abramenal** [comment]
Hi @Ckrill, thanks for quick feedback. Have you tried it with `v5.0.4` that was released yesterday?

**Comment by Ckrill** [comment]
Yes, I just tried that one.

**Comment by abramenal** [comment]
@Ckrill I'm struggling with reproducing this issue – are you (or someone else) able to set up a minimal repo with this issue?

**Comment by Ckrill** [comment]
Hi @abramenal we are not currently using the official Cypress package. Due to its lack of support for Webpack 5, we are using the one referenced in this [github issue comment](https://github.com/cypress-io/cypress/issues/8900#issuecomment-726066997).
I think the first step for us is to upgrade to the new Cypress 7, that now has support for Webpack 5.
If we continue to have the issue, I can try to set up a minimal repo, and let you know.
How does that sound?

**Comment by abramenal** [comment]
👌 makes sense
Please note there is this issue opened against cypress7 https://github.com/abramenal/cypress-file-upload/issues/289
I am going to prep a fix for it soon

**Comment by abramenal** [comment]
Related to #301

**Comment by abramenal** [comment]
Released in `v5.0.7`

**Comment by Ckrill** [comment]
Hey @abramenal,
I just wanted to let you know that I finally had time to test the new version. It looks like it is running smoothly now in our solution with Webpack 5.
Thanks for fix, good job!
